client/jetpack-connect/controller.js

```diff
@@ -88,6 +88,30 @@ const getPlanSlugFromFlowType = ( type, interval = 'yearly' ) => {
 	return get( planSlugs, [ interval, type ], '' );
 };
 
+/**
+ * !! Private !!
+ *
+ * Lazy initialized authorization query parser.
+ */
+let _authQueryParser = null;
```

Comment:
If we want to keep this private, we could move it inside `parseAuthorizationQuery` as a static variable. Could also help remove the leading underscore, and the need to have an initial ` = null` declaration.